src/travertino/declaration.py

```diff
@@ -143,10 +160,53 @@ def __delete__(self, obj):
         else:
             obj.apply(self.name, self.initial)
 
+    @property
+    def _name_if_set(self):
+        return getattr(self, "name", "")
+
+    def validate(self, value):
+        try:
+            return self.choices.validate(value)
+        except ValueError:
+            raise ValueError(
+                f"Invalid value {value!r} for property {self._name_if_set}; "
+                f"Valid values are: {self.choices}"
+            )
+
     def is_set_on(self, obj):
         return hasattr(obj, f"_{self.name}")
 
 
+class list_property(validated_property):
+    def validate(self, value):
+        if isinstance(value, str) or not isinstance(value, Sequence):
+            raise TypeError(
+                f"Value for list property {self._name_if_set} must be a non-string "
+                "sequence."
+            )
```

Comment:
It took me a moment to work out what this error was suggesting - I initially thought "wait... shouldn't it be a sequence of strings?" until I worked out you were referring to using a bare string as the value - which *is* a sequence, but isn't a valid input.

But... couldn't it be? I'm not 100% sold either way, but following "liberal in what you accept, conservative in what you return", we could convert a bare string into a list of one item. This would be consistent with directional properties accepting both `10` and `(10,)` (with both being normalised to `(10, 10, 10, 10)`.

Reply:
You know, that's what I initially intended to do, and I only veered away to be parallel to (what will be) the behavior of the `Font` class. (As you suggested, for backwards compatibility it *keeps* a single font family value if given, rather than storing it in a list. Otherwise it would break current versions of Toga.)

Now that I think about it though, while a  difference between the two might potentially cause confusion once in a while, it's probably worth it for a more intuitive interface on the one people will probably be using 90% of the time. Thoughts?

Reply:
I'm not sure I see the backwards compatibility problem here. Travertino's Font is a "dumb container"; it can hold either a single string or a list of strings.

The possible combinations then become:
* Old Toga, new Travertino - won't be using list_property for font family, so only a single string can be used anyway, and Toga's Font will continue to expect that.
* New Toga, old Travertino - can't happen as long as we set a new lower pin on the version
* New Toga, new Travertino - will (presumably) use list_property for font_family, so it will start allowing multiple values; but at the same time, the Toga Font implementation can be updated to *expect* multiple values.

In the last case, there will be existing code that says `Pack(font_family="Helvetica")`, so that needs to continue to be acceptable - so accepting a single string would seem to be *required* for compatibility.

What am I missing?

Reply:
Ohhhh! 💡

What I was missing was Toga's `Font`. I was envisioning an end user interacting with both `Pack.font` and `travertino.Font`, but they'd probably only ever be using the `Font` in Toga.

Reply:
> In the last case, there will be existing code that says Pack(font_family="Helvetica"), so that needs to continue to be acceptable - so accepting a single string would seem to be required for compatibility.

Damn, I was missing this too. Thank you for your patience!

Reply:
... and probably not even then. The Toga Font class *exists*, but it's almost entirely an internal detail. As a user, the only time you actually create a font instance is when you're using a Canvas; the only other time you use the Font class is to register a new user-provided font.